doc/docs.md

```diff
@@ -1285,19 +1251,16 @@ You can see a complete example for using C code in a V wrapper module here:
 You can use `-cflags` to pass custom flags to the backend C compiler. You can also use `-cc` to change the default C backend compiler.
 For example: `-cc gcc-9 -cflags -fsanitize=thread`.
 
-Ordinary zero terminated C strings can be converted to V strings with `string(cstring)` or `string(cstring, len)`.
+Ordinary zero-terminated C strings can be converted to V strings with `string(cstring)` or `string(cstring, len)`.
 
-NB: `string/1` and `string/2` do NOT create a copy of the `cstring`, so you should NOT free it after calling `string()`. If you need to make a copy of the C string (some libc APIs like `getenv/1` pretty much require that, since they
-return pointers to internal libc memory), you can use: `cstring_to_vstring(cstring)`
-
-On Windows, C APIs often return so called `wide` strings (utf16 encoding).
+NB: `string/1` and `string/2` do NOT create a copy of the `cstring`, so you should NOT free it after calling `string()`. If you need to make a copy of the C string (some libc APIs like `getenv/1` pretty much require that since they return pointers to internal libc memory), you can use: `cstring_to_vstring(cstring)`. On Windows, C APIs often return called `wide` strings (utf16 encoding).
```

Comment:
`On windows` should be on a separate line as it was, since it is a separate point.